Hunting Queries/Corelight/CorelightFilesSeen.yaml

```diff
@@ -0,0 +1,25 @@
+id: 0c3ad86f-6b92-4ff5-8d41-86ff7e01d518
+name: Corelight - Files in logs
+description: |
+  'Query searches for files which where seen in logs over period of time.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated